docs/source/make_and_open_docs.py

```diff
@@ -0,0 +1,9 @@
+from pathlib import Path
+import os
+import webbrowser
+
+path_makefile= Path().home() / "projects/manim-community/docs"
+os.system(f"cd {path_makefile} && make html")
+
+website = Path().home() / "projects/manim-community/docs/build/html/index.html"
+webbrowser.get('firefox').open_new_tab(f"{website}")
```

Comment:
```suggestion
webbrowser.open_new_tab(f"{website}")
```
I think that it is better to open just the default web-browser, since, if the user doesn't use FireFox (like me), the script won't work.

Reply:
okay, I will change this.
My intention was the following:
For normal web-browsing, I am using chrome.
To separate this from the docs-building workflow, I chose a browser that is not my default browser. Is it ok for you, when I add this line in the end but commented out?
Then people can quickly change to this option when they want to.

Reply:
Sure, that's fine for me.